CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -62,9 +62,9 @@ def parseArguments():
                         help="If passed, information will be returned about the exact type of output value "
                              "mismatch that occurs.",
                         action="store_true", default=False, dest="outputMismatchTypesFlag")
-    parser.add_argument("--format-check",
+    parser.add_argument("--skip-format-check",
                         help="If passed, text formatting will be checked as well as output values.",
-                        action="store_true", default=False, dest="formatCheckFlag")
```

Comment:
I think, you could also changethis to `store_false`and you wouldn't have to change the dest variable name or the logic later. two negatives `not skip` is usually harder to read/reason about

Reply:
going to leave this as-is.